trunk/conf/srt2rtc.conf

```diff
@@ -38,12 +38,15 @@ rtc_server {
 
 # @doc https://github.com/ossrs/srs/issues/1147#issuecomment-577607026
 vhost __defaultVhost__ {
+    srt {
+        enabled     on;
+    }
     rtc {
         enabled     on;
         # @see https://ossrs.net/lts/zh-cn/docs/v4/doc/webrtc#rtmp-to-rtc
-        rtmp_to_rtc off;
+        rtmp_to_rtc on;
```

Comment:
Please note that when coverting SRT to WebRTC, SRS actually converts SRT to RTMP, then converts RTMP to WebRTC, so we must enable `rtmp_to_rtc` here.